gendata/gendata.go

```diff
@@ -71,19 +72,34 @@ func reorderFieldStmts(tableStmt *tableStmt, fieldStmts []string) []string {
 	for _, field := range tableStmt.keyFields {
 		for i, fieldStmt := range fieldStmtsCopy {
 			if strings.HasPrefix(fieldStmt, fmt.Sprintf("`%s`", field)) {
-				orderedFieldStmts = append(orderedFieldStmts, fieldStmt)
+				orderedFieldStmts = append(orderedFieldStmts, withAggType(fieldStmt, ""))
 				fieldStmtsCopy = append(fieldStmtsCopy[:i], fieldStmtsCopy[i+1:]...)
 				break
 			}
 		}
 	}
 
+	// Add agg type for non-key cols if key type is agg
+	for i, fieldStmt := range fieldStmtsCopy {
+		aggType := ""
+		if tableStmt.keyType == KeyTypeAggregate {
+			aggType = randAggType()
+		}
+		fieldStmtsCopy[i] = withAggType(fieldStmt, aggType)
+	}
+
 	// append remain non-key cols at last
 	orderedFieldStmts = append(orderedFieldStmts, fieldStmtsCopy...)
 
 	return orderedFieldStmts
 }
 
+var aggTypes = []string{"MAX", "MIN", "REPLACE" /* do not support "SUM", because only numeric type can use it */}
+
```

Comment:
1. move to consts.go, then we just change it here later
2. append `REPLACE_IF_NOT_NULL ` or not? 
> https://doris.apache.org/zh-CN/docs/data-table/data-model
3. support `SUM` later?

Reply:
We will impl 2, 3 later when have time.

Reply:
> We will impl 2, 3 later when have time.

ok, get.